src/test/java/com/klarna/hiverunner/HiveRunnerExtensionTest.java

```diff
@@ -29,7 +29,7 @@
 @ExtendWith(HiveRunnerExtension.class)
 public class HiveRunnerExtensionTest {
 
-  @HiveSQL(files = {"HiveRunnerExtensionTest/test_query.sql"})
+  @HiveSQL(files = { "HiveRunnerExtensionTest/test_query.sql" })
```

Comment:
I don't think we should have the whitespace on either side of that string, we don't do that elsewhere in this project AFAIK.

Reply:
We don't, but the main branch has them in this file so I wanted to make both branches exactly the same. I'll take them off here but there's still two whitespaces in the main branch. If you want us to be pedantic I can make a PR to take those two whitespaces off the main branch :)

Reply:
I want us to be pedantic so yes please ;P